cc-pk-service/.env.example

```diff
@@ -0,0 +1,5 @@
+PORT=5000
+username="{YOUR USERNAME FOR PURKUKARTOITUS}"

```

Comment:
naming convention for constant variables: ALL_CAPS
also, could specify these more, example: PK_USERNAME, PK_PASSWORD ....
I had already setup username for my local node environment, so that got used instead of the username I defined here